### PR TITLE
Add "AR for the web" article.

### PR DESCRIPTION
**Pull request by jpmedley**
**Target Live Date:** 2018-06-13

- [x] This has been reviewed and approved by (John Pallet)
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+{# wf_blink_components: Blink>WebVR #}
+{# wf_featured_image: /web/updates/images/2018/06/sunflowers.png #}
+{# wf_featured_snippet: Augmented reality allows placement and tracking of virtual objects in a real-world view. #}
+
+# Augmented reality for the web {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In Chrome 67,
+[we announced](../05/welcome-to-immersive)
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: please don't use relative URLs, stick with absolute URLs (`/web/updates/2018/05/welcome-to-immersive`)

Comment (jpmedley) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: Done.

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+  if (xrPose && xrPose.poseModelMatrix) {
+    // Do the hit test and draw the reticle.
+  }
+}
+```
+
+Once you have the session and the pose, determine where the ray is casting. The
+[sample code uses](https://github.com/immersive-web/webxr-samples/blob/master/proposals/phone-ar-hit-test.html#L228)
+the
+[gl-matrix math library](https://www.google.com/url?q=https://github.com/toji/gl-matrix&sa=D&ust=1528822355331000&usg=AFQjCNH3Zq_kFrDLMdurQIeryUTWsel27w).
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: Remove the `google.com` redirect link from here

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+This is a good time to try out
+[the new code sample](https://immersive-web.github.io/webxr-samples/proposals/phone-ar-hit-test.html)
+in Chrome Canary. Before doing anything, double-check that you [have the
+correct flags enabled](#what). Now load the sample and click "Start AR".
+
+Notice a few things. First, the speed meter which you may recognize from the
+other immersive samples shows 30 frame per second instead of 60. This is the
+rate at which the web page receives images from the camera.
+
+![image](insert_image_url_here)
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: Insert image here

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+description: Augmented reality allows placement and tracking of virtual objects in a real-world view.
+
+{# wf_updated_on: 2018-06-15 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_tags: immersive-web,webar #}
+{# wf_blink_components: Blink>WebVR #}
+{# wf_featured_image: /web/updates/images/2018/06/sunflowers.png #}
+{# wf_featured_snippet: Augmented reality allows placement and tracking of virtual objects in a real-world view. #}
+
+# Augmented reality for the web {: .page-title }
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: Title Caps for Titles?

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+In Chrome 67,
+[we announced](/web/updates/2018/05/welcome-to-immersive)
+the WebXR Device API for both augmented reality (AR) and virtual reality (VR),
+though only the VR features were enabled. VR is an experience based purely on
+what's in a computing device. AR on the other hand allows you to render virtual
+objects in the real world. To allow placement and tracking of those objects, we
+just added the
+[WebXR Hit Test API](https://github.com/immersive-web/hit-test/blob/master/explainer.md)
+to Chrome Canary, a new method that helps immersive web code place objects in
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: Do you want to mention what version of Chrome that is?

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+[immersive web article](/web/updates/2018/05/welcome-to-immersive):
+"It's just the web."
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="1t1gBVykneA"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+"What Chrome extension do I need to install?" "There's no extension. It's just
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: I might change these to:
```
I had several questions at I/O that all had the same answer:

* What Chrome extension do I need to install?
* Do I need a special browser?
* What app do I need to install?

The answer to all was: **it's just the web**, it doesn't need any extensions
apps or anything other than your browser.
```

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+
+"What app do I need to install?" "There is no special app. It's just the web."
+
+This may be obvious to you since you're reading this on a website devoted to
+the web. If you build demonstrations with this new API, prepare for this
+question. You'll get it a lot.
+
+Speaking of IO, if you want to hear more about the immersive web in general,
+where it is, where it's going check
+[this video](https://www.youtube.com/watch?v=1t1gBVykneA&feature=youtu.be).
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: `out the video [the web is immersive](https://www.youtube.com/watch?v=1t1gBVykneA)`

Be specific about what the link is to.

**Comment by jpmedley** [comment]
PTAL ("Pete, take another look")

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+  session.requestAnimationFrame(onXRFrame);
+}
+```
+
+Though not as obvious in the hit test sample, you still need to
+[loop through the views](/web/updates/2018/05/welcome-to-immersive#views)
+to draw the scene. Drawing is done using WebGL APIs. You can do that if you're
+really ambitious. Though, we recommend using a framework. The immersive web
+samples use one created just for the demos called
+[Cottontail,](https://github.com/immersive-web/webxr-samples/tree/master/js/cottontail)
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: Comma in the link title

File: src/content/en/updates/2018/06/ar-for-the-web.md [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+### Placing an object
+
+An object is placed in AR when the user taps the screen. For that you use the
+`select` event. The important thing in this step is knowing where to place it.
+Since the moving reticle gives you a constant source of hit tests, the simplest
+way to place an object is to draw it at the location of the reticle at the last
+hit test. If you need to, say you have a legitimate reason not to show a
+reticle, you can call `requestHitTest()` in the select event. The immersive web
+sample
+[shows this](https://github.com/immersive-web/webxr-samples/blob/master/proposals/phone-ar-hit-test.html#L187).
```
Comment (petele) [inline src/content/en/updates/2018/06/ar-for-the-web.md]: Better description for link title please

File: src/content/en/updates/2018/06/ar-for-the-web.md (hunk 4 above)
Comment (jpmedley) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: I thought DevSite titles were sentence case now.

File: src/content/en/updates/2018/06/ar-for-the-web.md (hunk 5 above)
Comment (jpmedley) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: Apparently, no. The team intends to keep it canary only for next few cycles.

File: src/content/en/updates/2018/06/ar-for-the-web.md (hunk 6 above)
Comment (jpmedley) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: I like it as dialog. Adds a little variety.

File: src/content/en/updates/2018/06/ar-for-the-web.md (hunk 4 above)
Comment (petele) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: OK, haven't been enforcing that. Oops.

File: src/content/en/updates/2018/06/ar-for-the-web.md (hunk 5 above)
Comment (petele) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: Sounds good.

File: src/content/en/updates/2018/06/ar-for-the-web.md (hunk 6 above)
Comment (petele) [reply src/content/en/updates/2018/06/ar-for-the-web.md]: Sure

**Pull request merged**